docs/crd_mysql_cluster.md

```diff
@@ -0,0 +1,61 @@
+
+MySQLClusterSpec
+----------------
+
+| Field                         | Type                    | Required | Description                                                                                                                                                           |
+| ----------------------------- | ----------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `replicas`                    | int                     | No       | The number of instances. Available values are 1, 3, and 5. Default value is 1.                                                                                        |
+| `podTemplate`                 | [PodTemplateSpec]       | No       | `Pod` template for MySQL server container.                                                                                                                            |
+| `volumeClaimTemplate`         | [PersistentVolumeClaim] | No       | `PersistentVolumeClaim` template for MySQL server container.                                                                                                          |
+| `serviceTemplate`             | [ServiceSpec]           | No       | `Service` template for endpoints of MySQL server containers.                                                                                                          |
+| `mySQLConfigMapName`          | string                  | No       | `ConfigMap` name of MySQL config.                                                                                                                                     |
+| `rootPasswordSecretName`      | string                  | Yes      | `Secret` name for root user.                                                                                                                                          |
+| `replicationSourceSecretName` | string                  | Yes      | `Secret` name which contains replication source info. Keys must appear in [options].<br/> If this field is given, the `MySQLCluster` works as an intermediate master. |
+| `restore`                     | RestoreSpec             | No       | Spec to perform PiTR from existing cluster. If this field is filled, start restoring. This field is unable to be updated.                                             |
+
+RestoreSpec
+-----------
+
+| Field               | Type   | Required | Description                                                  |
+| ------------------- | ------ | -------- | ------------------------------------------------------------ |
+| `sourceClusterName` | string | Yes      | Source `MySQLCluster` name.                                  |
+| `pointInTime`       | [Time] | Yes      | Point-in-time of the state which the cluster is restored to. |
```

Comment:
Are these fields enough to find dump & binlog?
How can we find the object storage?

Reply:
Added a ref to ObjectStorage and a prefix (== dirname) to point the series of backup files.